packages/ui/src/components/Typography/title.tsx

```diff
@@ -5,7 +5,7 @@ import classNames from 'classnames'
 import { TYPOGRAPHY_CLASS_PREFIX } from './constants'
 
 declare const TITLE_LEVELS: [1, 2, 3, 4, 5, 6]
-declare const TITLE_TYPES: ['inverted']
+declare const TITLE_TYPES: ['inverted', 'secondary']
```

Comment:
I guess it's also could be danger / warning and etc to sync with text and paragraph, isn't it?